server/routes/helpdesk/validators/emailValidator.ts

```diff
@@ -1,5 +1,6 @@
 const EMAIL_PATTERN =
   /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+const CJSM_NET_EMAIL = /^.+[@.]cjsm\.net$/i
```

Comment:
I didn't realise we had 2 email validators now...

Reply:
Yeah, they are annoyingly different :(
This one validates you have entered an email address and that its not a CJSM address
The other one you are thinking of is the request a link validator. It validates you have entered an email address (any email address), and the validation to assert it is a CJSM address is at the API

Even if we could make the logic the same and bring them into the same class/function; the resultant error message text will be different 😢

Reply:
Don't these designers know anything about DRY? 🤣